src/cljs/main/broadfcui/main.cljs

```diff
@@ -112,6 +112,33 @@
                   "http://status.firecloud.org/"]
                  " for more information."])}))
 
+(defn status-alert [title message link]
+  [:div {:style {:borderBottom "1px solid" :borderBottomColor (:line-default style/colors)
+                 :backgroundColor (:exception-state style/colors) :padding "1rem"}}
+   [:div {:style {:color "#fff" :marginBottom "0.5rem" :fontWeight 600}}
+    [icons/ExceptionIcon {:size 18 :paddingRight "0.5rem"}] (if title title "Service Alert")]
+   [:div {:style {:color "#fff" :fontSize "90%"}}
+    [:div {} (str message " ")
+     (when link [:a {:href (str link) :target "_blank" :style {:color "#fff"}} "Read more..."])]]])
+
+(react/defc ShowStatusAlerts
+  {:render
+   (fn [{:keys [this state]}]
+     (let [alerts (:alerts @state)]
+       (when (not (empty? alerts))
+         [:div {}
+          (map (fn [alert]
+                 (status-alert (:title alert) (:message alert) (:link alert)))
+               alerts)])))
+   :component-did-mount
+   (fn [{:keys [state]}]
+     (utils/ajax {:url (config/alerts-json-url)
```

Comment:
how about this: as with all google calls, we should retry with exponential backoff and if we are still failing we should display a "There may be a problems accessing google cloud storage" and give them a link to https://status.cloud.google.com/